tests/test_geofile.py

```diff
@@ -1071,11 +1071,17 @@ def test_rename_column(tmp_path, suffix):
     assert "OPPERVL" not in result_layerinfo.columns
     assert "area" in result_layerinfo.columns
 
+    # Rename column with different casing
```

Comment:
Best add a column to the test file so you trigger the code that avoids an equal temp column being created...

Reply:
what do you mean? gfo.add_column first or just add a new column in the test file?

Reply:
Just use `gfo.add_column` to add it in the test...

Reply:
You can add the first tmp column in uppercase... then that has been tested as well... (e.g. TMP_0)